api/tests/integration/infrastructure/repositories/certification-repository_test.js

```diff
@@ -199,6 +199,21 @@ describe('Integration | Repository | Certification ', () => {
 
       await Promise.all(sessionWithStartedAndErrorCertifCourseIds.map(async (id) => expect((await get(id)).isPublished).to.be.false));
     });
+
+    it('does nothing when there are no certification courses', async () => {
+      // given
+      const sessionWithNoCertificationCourses = databaseBuilder.factory.buildSession({
+        certificationCenterId,
+        certificationCenter,
+      });
+      await databaseBuilder.commit();
+
+      // when
+      const result = await certificationRepository.publishCertificationCoursesBySessionId(sessionWithNoCertificationCourses.id);
+
+      // then
+      expect(result).to.not.exist;
```

Comment:
Not to throw?

Reply:
Ah oui c'est vrai que ça existe, je regarde

Reply:
J'ai fait un truc avec `await expect(promise).not.to.be.rejected`, je trouve ça moche aussi car si on enlève le `await` ça fonctionne beaucoup moins bien, 🤔

Reply:
Voir ici : [`d06c689` (#2582)](https://github.com/1024pix/pix/pull/2582/commits/d06c689202f7ff7a62f46cba1cbb36c019d1dd9b#diff-218c91ac5ba7d5911ab0bb)